src/EditorFeatures/CSharpTest/EditAndContinue/LineEditTests.cs

```diff
@@ -1417,6 +1417,50 @@ class C
                 Array.Empty<string>());
         }
 
+        [Fact]
+        public void Event_ExpressionBodyToBlock()
+        {
```

Comment:
I think "TopLevelEditingTests" is the right place for them.

---
In reply to: [128395722](https://github.com/dotnet/roslyn/pull/20931#discussion_r128395722) [](ancestors = 128395722)